### PR TITLE
Add jpeg writer

### PR DESCRIPTION
**Pull request by chakravarthi589**

### CODE REVIEW

**Comment by chakravarthi589** [comment]
@ajpmaclean : Please review.

**Comment by ajpmaclean** [comment]
Also please look through your remaining merge commits (326, 327, and 330), picking up on my comments here.

**Comment by chakravarthi589** [comment]
@ajpmaclean : Description updated in md file.

**Comment by ajpmaclean** [comment]
See my comment regarding hard coded file names

**Comment by chakravarthi589** [comment]
@ajpmaclean : am unable to see your comments.

**Comment by chakravarthi589** [comment]
image
I couldnt find your comments.

**Comment by ajpmaclean** [comment]
Ok I see what happened, my fault, you should be able to see comments now.

**Comment by chakravarthi589** [comment]
@ajpmaclean : I'll work on all your inputs and get back.

**Comment by ajpmaclean** [comment]
Here is a simple way of reading a line of optional parameters from the command line, similar to the C++ examples:
```Java
public class ReadCmdLine
{
  public static void main (String[] args)
  {
    // Provide default values.
    String fn = "Output.jpg";
    String some_ch = "";
    double some_dbl = 0.0;
    int some_int = 0;
    // We expect input as string string double int
    //
    for(int i = 0; i < args.length; ++i) {
      switch (i) {
        case 0:
          fn = args[i];
          break;
        case 1:
          some_ch = args[i];
          break;
        case 2:
          some_dbl = Double.parseDouble(args[i]);
          break;
        case 3:
          some_int = Integer.parseInt(args[i]);
          break;
      }
    }
    System.out.println(fn);
    System.out.println(some_ch);
    System.out.println(some_dbl);
    System.out.println(some_int);
  }
}
```

**Comment by chakravarthi589** [comment]
@ajpmaclean : Thank you for the snippet to read a line of optional parameters from the command line. Please review the changes now.

**Comment by chakravarthi589** [comment]
@ajpmaclean : With help of the snippet shared from you,
1. I have modified the program to accept a filename as command line argument else a default filename.
2. Have made use of vtkNamedColors. 

Please review the code.

File: src/Java/IO/JPEGWriter.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	  
+	  colors.SetColor("bkgColor", bkgColor);
+	  colors.SetColor("boxColor", boxColor);
+	  
+	  
+	// Provide default values.
+	String file_Name = "Output.jpg";
+	String File_Argument ="";
+	for(int i = 0; i < args.length; ++i)
+	{
```
Comment (ajpmaclean) [inline src/Java/IO/JPEGWriter.java]: Why do you need **File_Argument**? Use for **file_Name** use either camel case **fileName** or underscore all lower case **file_name**.

**Comment by chakravarthi589** [comment]
@ajpmaclean : Is it fine now? i have modified.

**Comment by ajpmaclean** [comment]
Thanks

**Pull request merged**